randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -831,14 +833,13 @@ Extra - asset_id: collision_camera_012
       All of the following:
           Lay Bomb or Lay Power Bomb
           Any of the following:
-              Space Jump
+              Gravity Suit or Space Jump
```

Comment:
Is a wall jump with Gravity enough to get up?

Reply:
Yes